lab-jinho/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "lab-jinho",
+  "version": "1.0.0",
+  "description": "",
+  "main": "gulpfile.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "DEBUG='restaurant*' mocha",
+    "start": "DEBUG='restaurant*' node server.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "Express": "^3.0.1",
```

Comment:
I'm not sure how this is a capital E, it should be 'express' because this will through a 'Cannot find express module' error when you run your tests.